AAP 1505--Update docs for using custom SSL certificates in OCP (#554)
RedHatInsights/red-hat-ansible-automation-platform-documentation: downstream/modules/platform/proc-controller-ldap-security.adoc

[id="proc_controller-ldap-security_{context}"]

= controller-ldap-security

> 

Reviewer: Same as the other topic above that you created, I think the title is missing and the file name got inserted here. Hopefully, we can use Newdoc in VSCode and this will take out any need to remember to add titles to our modules. The title should be appropriate for procedure modules (starting with a gerund, for example, "Configuring LDAP security for automation controller" or something along those lines).

> 

Reviewer: I would suggest adding a brief, concise abstract to this to help the user understand the "why" for this new topic. Might want to verify with the SME.

> . Under *LDAP Certificate Authority Trust Bundle* click the drop down menu and select a secret from the list.

Reviewer: ```suggestion
. Under *LDAP Certificate Authority Trust Bundle*, click the drop down list and select a secret.
```

> . Under *LDAP Password Secret* click the drop down menu and select a secret from the list.

Reviewer: ```suggestion
. Under *LDAP Password Secret*, click the drop-down list and select a secret.
```

> . Under *EE Images Pull Credentials Secret* click the drop down menu and select a secret from the list.

Reviewer: ```suggestion
. Under *EE Images Pull Credentials Secret*, click the drop-down list and select a secret.
```

> . Under *Bundle Cacert Secret* click the drop down menu and select a secret from the list.

Reviewer: ```suggestion
. Under *Bundle Cacert Secret*, click the drop-down list and select a secret.
```

> . Under *Service Type* click the drop down menu and select *ClusterIP*, *LoadBalancer*, or *NodePort*. 

Reviewer: ```suggestion
. Under *Service Type*, click the drop-down list and select one of the following options: 

- ClusterIP* 

- *LoadBalancer*

- *NodePort*. 

```
